bin/java-petstore-jersey2.sh

```diff
@@ -26,6 +26,8 @@ fi
 
 # if you've executed sbt assembly previously it will use that instead.
 export JAVA_OPTS="${JAVA_OPTS} -XX:MaxPermSize=256M -Xmx1024M -DloggerPath=conf/log4j.properties"
-ags="$@ generate -i modules/swagger-codegen/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml -l java -c bin/java-petstore-jersey2.json -o samples/client/petstore/java/jersey2"
+ags="$@ generate -i modules/swagger-codegen/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml -l java -c bin/java-petstore-jersey2.json -o samples/client/petstore/java/jersey2 -DdateLibrary=joda,hideGenerationTimestamp=true"
 
+rm -rf samples/client/petstore/java/jersey2/src/main
```

Comment:
I would suggest adding a line to echo a message, e.g. "Removing files and directories under samples/client/petstore/java/jersey2/src/main" so that the user/developer is aware that the script will purge the existing Java files.